typescript/agentkit/src/action-providers/cdp/cdpApiActionProvider.ts

```diff
@@ -99,10 +107,11 @@ from another wallet and provide the user with your wallet details.`,
   /**
    * Checks if the Cdp action provider supports the given network.
    *
-   * @param _ - The network to check.
+   * @param network - The network to check.
    * @returns True if the Cdp action provider supports the network, false otherwise.
    */
-  supportsNetwork = (_: Network) => true;
+  supportsNetwork = (network: Network) =>
+    network.protocolFamily === "evm" || network.networkId === "solana-devnet";
```

Comment:
`address_reputation` action is not supported on `solana-devnet". Can we call out that tighter network scoping is done at the action implementation